docs/content/usage/actions/act-runner.en-us.md

```diff
@@ -114,6 +114,12 @@ If you cannot see the settings page, please make sure that you have the right pe
 
 The format of the registration token is a random string `D0gvfu2iHfUjNqCYVljVyRV14fISpJxxxxxxxxxx`.
 
+A registration token can also be obtained from the gitea [command-line interface](../../../administration/command-line.en-us.md#actions-generate-runner-token):
```

Comment:
It should be `../../administration/command-line.en-us.md#actions-generate-runner-token`.